### PR TITLE
Add new EaR+KMS CI jobs

### PR DESCRIPTION
**Pull request by vponomaryov**
Commits:
- `ci(kms): add 1d longevity CI job for running EaR+KMS features`
- `ci(kms): add 6h multiDC longevity CI job for running EaR+KMS features`
- `ci(kms): add perf CI jobs for running EaR+KMS features`

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+]
+stress_cmd: [
+  "cassandra-stress write cl=QUORUM duration=360m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=30 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+  "cassandra-stress read  cl=QUORUM duration=360m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=30 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+]
+round_robin: true
+
+n_db_nodes: '3 3'
+n_loaders: '1 1'
+region_aware_loader: true
```
Comment (fruch) [inline test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml:15]: we should remove it

File: test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+n_monitor_nodes: 1
+
+instance_type_db: 'i3en.xlarge'
```
Comment (fruch) [inline test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml:18]: let pick a small instance, maybe i4i.2xlarge ?

**Review by fruch — CHANGES_REQUESTED** [review]
just changing the instance on multi-dc

File: test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml (hunk 1 above)
Comment (vponomaryov) [reply test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml]: Done

File: test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml (hunk 2 above)
Comment (vponomaryov) [reply test-cases/longevity/longevity-encryption-at-rest-20GB-6h-multidc.yaml]: Done

**Comment by vponomaryov** [comment]
@fruch it is waiting for you?

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**